matchzoo/embedding.py

```diff
@@ -59,22 +59,20 @@ def build_matrix(
         self,
         term_index: typing.Union[
             dict, processor_units.VocabularyUnit.TermIndex],
-        initializer=lambda: np.random.uniform(-0.2, 0.2)
+        initialize_range: tuple = (-0.2, 0.2)
```

Comment:
It's not always a uniform distribution. I usually use trunc normal, and I wanted to keep that flexibility.

Reply:
To pass an initializer, we would assign each element of the matrix, as is shown in lines 75-77 in original `embedding.py`. This is too much time-consuming.

Reply:
To pass an initializer, we would assign each element of the matrix, as is shown in lines 75-77 in original embedding.py. This is too much time-consuming.

Reply:
I have recovered the embedding initialization method to keep the flexibility.